model.md

```diff
@@ -0,0 +1,66 @@
+
+## Motivation
+
+Lack of interoperability of pedigree data between systems results in a substantial double-entry burden for already-overloaded nurses, genetic counselors, and support staff, and introduces the potential for serious errors.
+
+# Classes
+
+![image](https://user-images.githubusercontent.com/4251264/105409205-1c53b200-5bfe-11eb-9d77-ed5630138eaf.png)
+
+## Pedigree
+
+A `Pedigree` is collection of selected information about a family, including the individuals, relationships between them, and relevant health conditions.
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| proband | ID | optional | id of Individual that is the index case for the family |
+| consultand | ID | optional | id of Individual that is the focus of the current analysis |
+| collectedAt | Date | optional | the date the pedigree was collected, as ISO full or partial date, i.e. YYYY, YYYY-MM, or YYYY-MM-DD |
+| reasonCollected | Concept | optional | the health condition being investigated in the family; if any Individual has the `affected` property defined, it refers to this condition |
+
+## Individual
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| id | ID | required | logical id |
+| name | string | optional | name of the individual |
+| identifiers | list of string | optional | external identifiers for the individual |
+| sex | Concept | optional | Sex for Clinical Use, values: `male`, `female`, `other`, `unknown`; See: https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions and https://www.hl7.org/fhir/valueset-administrative-gender.html |
+| gender | Concept | optional | Presumed or reported gender identity, values: `male`, `female`, `non-binary`, `non-disclosed`, `trans`, ...; See: https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions |
+| lifeStatus | Concept | recommended | The presumed/accepted life status of the individual as of the pedigree collection date; one of: `alive`, `deceased`, `unborn` |
+| affected | boolean | not recommended | whether or not the individual is affected by the condition being investigated in this pedigree; included for PED backwards compatibility |
+
+## Relationship
+
+### properties
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| id | ID | required | logical id |
+| individual | ID | required | identifier of the subject Individual; equivalent to the FHIR "Scoping Individual" and the Biolink "Object" |
+| relative | ID | required | identifier of the relative Individual; equivalent to the FHIR "Player" and the Biolink "Subject" |
+| relation | Concept | required | the relationship between `individual` and `relative`; if the relationship is directed, it is the relationship that the relative has to the individual (e.g., individual is the child, relative is the mother, then relation is parent); terms should come from the relationship terminology, below |
+
+## Relationship Terminology
```

Comment:
Where any directionality exists in the relationships, it is... 
  1. Not obvious from the label e.g. 'parent' would read better as 'parent_of'
  1. Reads backwards compared to the more standard subject-predicate-object e.g. 'child' <- 'parent' - 'mother'  compared to mother - parent_of -> child
 
@pnrobinson - do you have any suggestions?

Reply:
We could present the field order as "relative", "relation", "individual" to match subject-predicate-object. Happy to revise the labels for all proposed terms to have "of" in them as well if that's more standard.

Reply:
I got some feedback from another ontology expert and they suggested the terminology be structured as a hierarchy of object properties of the form: `hasX`